pkgs/servers/firebird/4.nix

```diff
@@ -0,0 +1,59 @@
+{ autoreconfHook
+, fetchFromGitHub
+, icu
+, lib
+, libedit
+, libtomcrypt
+, libtommath
+, stdenv
+, unzip
+, zlib
+
+, superServer ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "firebird";
+  version = "4.0.2";
+
+  src = fetchFromGitHub {
+    owner = "FirebirdSQL";
+    repo = "firebird";
+    rev = "v${version}";
+    sha256 = "sha256-hddW/cozboGw693q4k5f4+x9ccQFWFytXPUaBVkFnL4=";
+  };
+
+  configureFlags = [
+    "--with-system-editline"
+  ] ++ (lib.optional superServer "--enable-superserver");
+
+  LD_LIBRARY_PATH = lib.makeLibraryPath [ icu ];
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  buildInputs = [
+    icu
+    libedit
+    libtomcrypt
+    libtommath
+    unzip
+    zlib
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    cp -r gen/Release/firebird/* $out
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "SQL relational database management system";
+    downloadPage = "https://github.com/FirebirdSQL/firebird/";
+    homepage = "https://firebirdsql.org/";
+    changelog = "https://github.com/FirebirdSQL/firebird/blob/master/CHANGELOG.md";
+    license = [ "IDPL" "Interbase-1.0" ];
```

Comment:
Those should be moved to lib.licenses

Reply:
This license is not used anywhere else I know of. Can we just leave this as it is? 

* https://www.firebirdsql.org/en/initial-developer-s-public-license-version-1-0/
* https://www.firebirdsql.org/en/interbase-public-license/

Reply:
Just to double check: are those licenses considered free?

Reply:
This license is essentially a rebranded version of MPL-1.1, but with some important changes. A new AMENDMENTS section is introduced, which includes an advertisement clause, non-endorsement and no-trademark clauses, as well as product/project naming rules. These changes are summarised in the new §6.4 where it is also explicitly stated that the license is a modificaton on MPL-1.1.

https://spdx.org/licenses/Interbase-1.0.html